packages/appstore/src/helpers/total-assets-helper.ts

```diff
@@ -0,0 +1,34 @@
+import { useCFDAccounts, useExchangeRate, usePlatformAccounts } from '@deriv/hooks';
```

Comment:
Appstore package is the current Trader's Hub.
Surely a rename is needed.

Reply:
@wojciech-deriv it's just a helper function, I needed it in the component, and it was better to just move it in the helper folder instead of keeping it inside of the `AssetSummary` component file